docs/enroute/Brisbane Centre/INL.md

```diff
@@ -90,12 +90,12 @@ The Standard Assignable level for YBSU arrivals from BN TCU to **NSA** is `F130`
 Refer to [Brisbane TCU Airspace Division](../../../terminal/brisbane/operations/#airspace-division) for information on airspace divisions when **BAS**, **BDN** and/or **BDS** are online.
 ### Enroute
 
-As per [Standard coordination procedures](../../../controller-skills/coordination/#enr-enr), Voiceless, no changes to route or CFL within **20nm** to boundary.
+As per [Standard coordination procedures](../../../controller-skills/coordination/#enr-enr), Voiceless, no changes to route or CFL within **50nm** to boundary.
 
 ### INL Internal
-As per [Standard coordination procedures](../../../controller-skills/coordination/#enr-enr), Voiceless, no changes to route or CFL within **20nm** to boundary.
+As per [Standard coordination procedures](../../../controller-skills/coordination/#enr-enr), Voiceless, no changes to route or CFL within **50nm** to boundary.
```

Comment:
I left these internal coord parameters at 20nm so people could make changes closer to the boundary for sequencing/descent purposes, like BLA->ELW and HYD->PIY. Otherwise technically you may have to do coord every time you assign descent on a STAR

Reply:
> I left these internal coord parameters at 20nm so people could make changes closer to the boundary for sequencing/descent purposes, like BLA->ELW and HYD->PIY. Otherwise technically you may have to do coord every time you assign descent on a STAR

Ah right, I was just trying to close this issue: https://github.com/vatpac-technology/sops/issues/169

I'll put them back.